ubuntu/touch-symlinks.conf

```diff
@@ -0,0 +1,11 @@
+start on ((filesystem
```

Comment:
This script is not needed, the symlinks are placed in https://github.com/ubports/android_device_lge_hammerhead/blob/ef33fc1d9484dc8045b741b9711cc2f1b2126528/init.hammerhead.rc#L68

Reply:
Yeah it init.hammerhead.rc as it, but also on 5.1 and we also added this file.